gpjax/sklearn/optim.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+import tensorflow_probability.substrates.jax as tfp
+
+from gpjax.kernels import AbstractKernel
+from gpjax.mean_functions import AbstractMeanFunction
+from gpjax.sklearn.base import BaseEstimator
+
+tfd = tfp.distributions
+
+
+@dataclass
+class GPJaxOptimizer(BaseEstimator):
+    kernel: AbstractKernel
+    mean_function: AbstractMeanFunction = None
+    n_inducing: int = -1
```

Comment:
What is the intended usage for the `GPJaxOptimizer`? (Why do we have it?)

Reply:
Once we’re happy with an API, I see it being analogous to Scipy’s minimise fn